tests/framework/Samplers/Restart/test_restart_stochpoly.xml

```diff
@@ -47,12 +47,6 @@
       <Output class="OutStreams" type="Print">restart</Output>
       <Output class="OutStreams" type="Print">fine</Output>
     </IOStep>
-    <IOStep name="printROM">
-      <Input class="Models" type="ROM">romRestart</Input>
-      <Input class="Models" type="ROM">romFine</Input>
-      <Output class="OutStreams" type="Print">romRestart</Output>
-      <Output class="OutStreams" type="Print">romFine</Output>
-    </IOStep>
```

Comment:
@PaulTalbot-INL ?

Reply:
The rom output was not tested so I removed it (since this step failed).  An alternative fix would be to modify the input so the rom print is generated and then test it properly.

Reply:
Josh and I did talk about this; the output of the ROM was not tested, so it can be removed for this test. It is tested in other tests.